src/robot/libraries/String.py

```diff
@@ -403,13 +441,25 @@ def get_lines_matching_regexp(
         match = regexp.search if partial_match else regexp.fullmatch
         return self._get_matching_lines(string, match)
 
-    def _get_matching_lines(self, string, matches):
+    # TODO(@pekkaklarck): some paramters could be string, some bytes -> risk of incompatibilies
```

Comment:
This isn't a keyword so no need to mention this anywhere. The final typing depends on what we want to do with the above keywords.